millicore.go

```diff
@@ -5,6 +5,10 @@ import (
 	"path/filepath"
 )
 
+// GetMillicoreConfigTasks will create a function matching the CheckTask
+// interface for each project to retrieve the millicore config in that
```

Comment:
I'm not sure either, but looking around, it seems to be referred to as Millicore, so I will change here.